sparse/core.py

```diff
@@ -2364,25 +2412,23 @@ def maybe_densify(self, allowed_nnz=1000, allowed_fraction=0.25):
         >>> np.allclose(x, s.todense())
         True
 
-        You can also specify the minimum allowed sparsity or the maximum number
-        of nonzero values. If both conditions are unmet, this method will throw
+        You can also specify the minimum allowed density or the maximum number
+        of output elements. If both conditions are unmet, this method will throw
         an error.
 
         >>> x = np.zeros((5, 5), dtype=np.uint8)
         >>> x[2, 2] = 1
         >>> s = COO.from_numpy(x)
-        >>> s.maybe_densify(allowed_nnz=5, allowed_fraction=0.25)
+        >>> s.maybe_densify(max_size=5, min_density=0.25)
         Traceback (most recent call last):
             ...
         NotImplementedError: Operation would require converting large sparse array to dense
```

Comment:
Line 2425...

```
NotImplementedError: Operation would require converting large sparse array to dense
```

Need to change this to `ValueError`.

Reply:
It's causing doctests to fail. :)

Reply:
Currently there are a million tests failing on `master`, it is hard to spot my own mistakes...

Reply:
That's strange, [Travis shows everything is working on master](https://travis-ci.org/mrocklin/sparse/branches).

Reply:
I checked, they only fail on Numpy 1.12. Latest Numpy works fine...

Reply:
That is really weird, I tested with Numpy 1.13.3 on my local system and it still works fine.